src/app/components/arlas-wui-root/arlas-wui-root.component.scss

```diff
@@ -31,17 +31,37 @@
     padding: 0;
   }
 
-  ::ng-deep.current-coordinate,
-  ::ng-deep.current-coordinate-edition {
-    right: calc($sm-spacing + $map-scale-max-width + $default-spacing);
-    bottom: calc($timeline-height + $sm-spacing);
-  }
+  .arlas-map {
+    display: flex;
+    width: 100%;
+    height: 100%;
 
-  .map-container-tight-coordinates {
     ::ng-deep.current-coordinate,
     ::ng-deep.current-coordinate-edition {
-      right: $sm-spacing;
-      bottom: calc($timeline-height + $sm-spacing + $map-scale-height + $sm-spacing);
+      right: calc($sm-spacing + $map-scale-max-width + $default-spacing);
```

Comment:
the need of ng-deep makes me think these classes should be in the new map component no ?

Reply:
Here for the current coordinate, it is a class that comes from arlas-web-components, it already was like this before.